lms/djangoapps/courseware/views/views.py

```diff
@@ -1635,6 +1635,12 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True, disable_sta
                 if not _check_sequence_exam_access(request, seq_block.location):
                     return HttpResponseForbidden("Access to exam content is restricted")
 
+        # AI Translations feature for course content, passes a translation language to blocks
+        # which can call through the ai_translation service (passed through XBlock runtime)
+        # to receive translated versions of course content.
+        if whole_course_translations_enabled_for_course(course_key):
+            student_view_context['translate_lang'] = request.GET.get('translate_lang')
```

Comment:
Nice! I was wondering where this would live.

Reply:
I still don't know if this is the best place, but it is ,probably, the smallest change.